src/components/FloatingDropdown/FloatingDropdown.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+
+import { Dropdown as CoreDropdown } from 'wix-ui-core/dropdown';
+
+import { DropdownOption } from '../Dropdown/DropdownOption';
+import { TPAComponentsConsumer } from '../TPAComponentsConfig';
+import styles from './FloatingDropdown.st.css';
+import { FloatingDropdownBase } from './FloatingDropdownBase';
+import { DATA_HOOKS } from './constants';
+import { FloatingDropdownOptionProps } from './FloatingDropdownOption';
+
+export interface FloatingDropdownProps {
+  /** Defines a string value that labels the current element. Optional. */
+  'aria-label'?: string;
+  /** Identifies the element (or elements) that labels the current element. Optional. */
+  'aria-labelledby'?: string;
+  /** Applies disabled styles. Optional. Boolean. */
+  disabled?: boolean;
+  /** Label string value. Required. */
+  label: string;
+  /** Callback which is being called when value changes. Function. Optional. */
+  onChange?(selectedOption: FloatingDropdownOptionProps): void;
+  /** An array of dropdown option items. Option item structure: { id: string; value: string; isSelectable: boolean; } */
+  options: FloatingDropdownOptionProps[];
+  /** A placeholder which is being displayed when no value is selected. String. Optional. */
+  placeholder?: string;
+  /** An id of initially selected item. String. Optional. */
+  value?: string;
+  /** Force dropdown open. Use for visual test. Optional. Boolean. */
+  forceContentElementVisibility?: boolean;
+}
+
+type DefaultProps = Required<
+  Pick<FloatingDropdownProps, 'disabled' | 'options'>
+>;
+
+interface State {
+  selectedOption: FloatingDropdownOptionProps;
```

Comment:
This ones my fault:
I have a PR I haven't merged yet
It removes the state from the dropdown.
Maybe you can do it here as well.
You can see my pr here:
https://github.com/wix/wix-ui-tpa/pull/283/files

It's because we had a bug that was caused because of this internal state

Reply:
@jonathanadler I made component stateless. 
After this change, documentation might be a bit confusing, as it does not reflect real world example of controlled component.

Reply:
You mean in the storybook?

Reply:
Yes, in storybook, https://wix-wix-ui-tpa-pr-313.surge.sh/